pkgs/development/python-modules/pysimplegui/default.nix

```diff
@@ -30,6 +31,7 @@ buildPythonPackage rec {
     description = "Python GUIs for Humans";
     homepage = "https://github.com/PySimpleGUI/PySimpleGUI";
     license = licenses.lgpl3Plus;
+    broken = stdenv.isDarwin;
```

Comment:
there should be a comment why the package is marked broken

Reply:
ok

Reply:
The comment you added does not really explain why. I just took a quick look at hydra where the package builds https://hydra.nixos.org/job/nixpkgs/trunk/python310Packages.pysimplegui.x86_64-darwin

Reply:
Well, this PR started from https://zh.fail/failed/by-maintainer/lucasew.html

Reply:
Is it possible it might be failing for python 3.11 but not for python 3.10?

Reply:
Gonna look It deeply tomorrow

Reply:
Yes, it only fails on 3.11 because tkinter is missing. We can mark the package for 3.11 and darwin broken.